examples/tv-app/android/App/platform-app/src/main/java/com/matter/tv/server/service/AppPlatformService.java

```diff
@@ -0,0 +1,114 @@
+/*
+ *   Copyright (c) 2021-2022 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package com.matter.tv.server.service;
+
+import android.app.Activity;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import androidx.annotation.NonNull;
+import com.matter.tv.server.MatterCommissioningPrompter;
+import com.matter.tv.server.handlers.ContentAppEndpointManagerImpl;
+import com.matter.tv.server.model.ContentApp;
+import com.matter.tv.server.receivers.ContentAppDiscoveryService;
+import com.matter.tv.server.tvapp.AppPlatform;
+
+public class AppPlatformService {
```

Comment:
Some brief doccomment is needed here:
 - why does this class exist
 - how and why does one use it?